site/_posts/2017-03-22-pycon-pune-2017.html.md

```diff
@@ -0,0 +1,43 @@
+
+## Brief about an Event :
+
+This was my second PyCon and first PyCon Pune Python Conference. I had a great time in this [PyCon Pune-2017](https://pune.pycon.org/). This year, PyCon Pune was held in Pune at [Amanora The Fern Hotels](https://goo.gl/maps/c6bMuVihxtq). PyCon Pune is a two day conference where all Pythonistas share their knowledge about Python, Django, Mailman etc. This year PyCon Pune 2017 had 6 Keynote speakers. Check out our [twitter feed](https://twitter.com/manageiq) and look for the [#pyconpune](https://twitter.com/search?q=%23pyconpune&src=typd) hashtag.
+
+There were several booths present at PyCon Pune 2017 which were representing their products/technologies. We had a ManageIQ pod which was part of the Red Hat booth and I was also a part of both at PyCon Pune 2017. People were interested in ManageIQ and wanted to know about *“What is ManageIQ…?”*.
+
+![](/assets/images/blog/PyConPune_location.gif)
+
+## Day One, 16 February, 2017 :
+
+On the first day of the conference, [Sachin Patil](https://github.com/psachin), [Aziza Karol](https://twitter.com/AzizaKarol), [Niyaz Ansari](https://twitter.com/ansari_niyaz), [Satyajit Bulage](https://twitter.com/s_satya4ever) explained what *ManageIQ* is about and how to start using *ManageIQ*. Throughout the day, there was a quiz contest going on which was based on *Python* as well as *ManageIQ*. The quiz contest got huge responses from various Pythonistas.
+
+## Day Two, 17 February, 2017 :
+
+On the second day of the conference, we changed the quiz format a little bit. We introduced *ManageIQ* to the audience with 15-20 mins description by [Prasad Mukhedkar](https://twitter.com/eprasad). [Neha Chough](https://twitter.com/Nehachugh18) later on asked questions based on same description. We received overwhelming responses for the second day’s quiz contest as well.
+
+[![](/assets/images/blog/MIQ_FB_2017_03.png)](https://twitter.com/i/moments/834117124872364033)
+
+## Q and  A :
+
+On both days, we gave the audience a hands-on using *ManageIQ* deployments on various topics such as *"Provisioning Virtual Machines and Instances"*, *"how SmartState Analysis gathers all required information from Virtual Machines and Instances"*, *"how SmartState Analysis can fetch data from various providers like users, groups, files, packages etc."*
+
+## Conclusion of an event :
+
+On that note, we had learned, enjoyed and tried to share as much information as possible about *ManageIQ* with a lot of enthusiastic people during the two days at [PyCon Pune 2017](https://pune.pycon.org/).
+
+Too many people were interested to get more information about *ManageIQ*, and they shared their contact information with us. In future, we will be planning meetups for them.
+
+## Wrapping up :
+
+This was a quick overview of what have we done in PyCon Pune-2017, but feel free to check out our event pics on [twitter feed](https://twitter.com/manageiq).
```

Comment:
Might be better to link to the twitter moment for pics, i.e. change 

`on [twitter feed](https://twitter.com/manageiq)`

to

`in this [twitter moment](https://twitter.com/i/moments/834117124872364033)`